src/viur/core/bones/text.py

```diff
@@ -349,27 +350,31 @@ def __init__(
         self,
         *,
         validHtml: Union[None, Dict] = __undefinedC__,
-        maxLength: int = 200000,
+        max_length: int = 200000,
         srcSet: Optional[Dict[str, List]] = None,
         indexed: bool = False,
         **kwargs
     ):
         """
             :param validHtml: If set, must be a structure like :prop:_defaultTags
             :param languages: If set, this bone can store a different content for each language
-            :param maxLength: Limit content to maxLength bytes
-            :param indexed: Must not be set True, unless you limit maxLength accordingly
+            :param max_length: Limit content to max_length bytes
+            :param indexed: Must not be set True, unless you limit max_length accordingly
             :param srcSet: If set, inject srcset tags to embedded images. Must be a dict of
                 "width": [List of Ints], "height": [List of Ints], eg {"height": [720, 1080]}
         """
+        # fixme: Remove in viur-core >= 4
+        if "maxLength" in kwargs:
+            warnings.warn("maxLength is deprecated, please use max_length")
```

Comment:
Sorry, my suggestion in the other PR wasn't complete, please add the class:
```suggestion
            warnings.warn("maxLength parameter is deprecated, please use max_length", DeprecationWarning)
```